packages/react-lowcode/src/codegen/generation/context/injection-context.ts

```diff
@@ -0,0 +1,17 @@
+import { CodegenNaming, Naming } from "../interfaces/naming";
+
+export class InjectionContext{
+    
+    constructor(){
+        this._naming = new CodegenNaming()
+    }
+
+    private _naming : Naming
+
+    public get naming() : Naming {
+        return this._naming;
+    }
+    private set value(naming : Naming) {
```

Comment:
Let's have immutable readonly objects whenever possible (please avoid using setters). Injection context si pretty good case for this. We can have just interface (and if needed some builder).